tmt/steps/execute/scripts/tmt.sh.j2

```diff
@@ -0,0 +1,4 @@
+# shellcheck shell=bash
+
+# tmt provides executable scripts under this path
+export PATH={{ SCRIPTS_DEST_DIR }}:$PATH
```

Comment:
Shouldn't the value be wrapped in quotes, to prevent nasty and ugly effects of spaces and other magic characters? `PATH="{{ ... }}:$PATH"`?

Reply:
```suggestion
export PATH="{{ SCRIPTS_DEST_DIR }}:$PATH"
```